src/commands/utils/clusters.ts

```diff
@@ -375,6 +375,19 @@ export async function getManagedCluster(
     }
 }
 
+export async function getKubernetesVersionInfo(
+    client: azcs.ContainerServiceClient,
+    location: string,
+    clusterName: string,
```

Comment:
Can we remove the `clusterName` parameter here? It's only used in the error message, which should say something like `Failed to retrieve versions for ${location}`

Reply:
Added a commit to address this